### PR TITLE
Update index.d.ts

### PR DESCRIPTION
**Pull request by iuyt9003**
Updating readme please complete me. didn't know from where to take the other info

### CODE REVIEW

**Comment by patrick-fs** [comment]
Hi @iuyt9003 thanks for the suggestion! Would you mind updating your doc with the following information? 

/**
 * FullStory Client SDK snippet options.
 *
 * - orgId: Reference for your [Org Id](https://help.fullstory.com/hc/en-us/articles/360047075853) listed in FullStory.
 * - namespace: Global object name that contains the FullStory browser API methods and properties. Defaults to `FS`.
 * - debug: Debug mode with extra browser console logging.
 * - host: The recording server host domain. Can be set to direct recorded events to a proxy that you host. Defaults to 'fullstory.com'.
 * - script: FullStory script host domain. FullStory hosts the `fs.js` recording script on a CDN, but you can choose to host a copy yourself. Defaults to `edge.fullstory.com`.
 * - recordCrossDomainIFrames: FullStory can record cross-domain iFrames. Certain limitations apply and can be [found here](https://help.fullstory.com/hc/en-us/articles/360020622514-Can-FullStory-capture-content-that-is-presented-in-iframes-#h_01F1G333PYKPGZ4B42WDBV3YKV). Defaults to `false`.
 * - recordOnlyThisIFrame: When set to `true`, FullStory records the iFrame as its own unique session. Defaults to `false`. Additional conditions apply and can be found [here](https://help.fullstory.com/hc/en-us/articles/360020622514-Can-FullStory-capture-content-that-is-presented-in-iframes-#h_01F1G33B40Q2TPQA8MA7SF8Y5P).
 * - devMode: In dev mode FullStory won't record sessions. Any calls to SDK methods will `console.warn` that FullStory is in `devMode`. Defaults to 'false'.
 */
 
 Once done, we can review and approve.

**Comment by iuyt9003** [comment]
Hey, @patrick-fs Thanks! now it looks much better :) had no idea about the host stuff. tried to dig but saw it's moving to internal stuff :)

File: src/index.d.ts [hunk 1]
```diff
@@ -1,11 +1,23 @@
+/**
+ * FullStory Client SDK snippet options.
+ *
+ * - orgId: Reference for your [Org Id](https://help.fullstory.com/hc/en-us/articles/360047075853) listed in FullStory.
+ * - namespace: Global object name that contains the FullStory browser API methods and properties. Defaults to `FS`.
+ * - debug: Debug mode with extra browser console logging.
+ * - host: The recording server host domain. Can be set to direct recorded events to a proxy that you host. Defaults to 'fullstory.com'.
+ * - script: FullStory script host domain. FullStory hosts the `fs.js` recording script on a CDN, but you can choose to host a copy yourself. Defaults to `edge.fullstory.com`.
+ * - recordCrossDomainIFrames: FullStory can record cross-domain iFrames. Certain limitations apply and can be found [here](https://help.fullstory.com/hc/en-us/articles/360020622514-Can-FullStory-capture-content-that-is-presented-in-iframes-#h_01F1G333PYKPGZ4B42WDBV3YKV). Defaults to `false`.
```
Comment (patrick-fs) [inline src/index.d.ts:9]:
```suggestion
 * - recordCrossDomainIFrames: FullStory can record cross-domain iFrames. Defaults to `false`. Certain limitations apply and can be found [here](https://help.fullstory.com/hc/en-us/articles/360020622514-Can-FullStory-capture-content-that-is-presented-in-iframes-#h_01F1G333PYKPGZ4B42WDBV3YKV). 
```

File: src/index.d.ts [hunk 2]
```diff
@@ -1,11 +1,23 @@
+/**
+ * FullStory Client SDK snippet options.
+ *
+ * - orgId: Reference for your [Org Id](https://help.fullstory.com/hc/en-us/articles/360047075853) listed in FullStory.
+ * - namespace: Global object name that contains the FullStory browser API methods and properties. Defaults to `FS`.
+ * - debug: Debug mode with extra browser console logging.
+ * - host: The recording server host domain. Can be set to direct recorded events to a proxy that you host. Defaults to 'fullstory.com'.
```
Comment (patrick-fs) [inline src/index.d.ts:7]:
```suggestion
 * - host: The recording server host domain. Can be set to direct recorded events to a proxy that you host. Defaults to `fullstory.com`.
```

File: src/index.d.ts [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
+ * - script: FullStory script host domain. FullStory hosts the `fs.js` recording script on a CDN, but you can choose to host a copy yourself. Defaults to `edge.fullstory.com`.
+ * - recordCrossDomainIFrames: FullStory can record cross-domain iFrames. Certain limitations apply and can be found [here](https://help.fullstory.com/hc/en-us/articles/360020622514-Can-FullStory-capture-content-that-is-presented-in-iframes-#h_01F1G333PYKPGZ4B42WDBV3YKV). Defaults to `false`.
+ * - recordOnlyThisIFrame: FullStory can record the iFrame as its own unique session. Defaults to `false`. Additional conditions apply and can be found [here](https://help.fullstory.com/hc/en-us/articles/360020622514-Can-FullStory-capture-content-that-is-presented-in-iframes-#h_01F1G33B40Q2TPQA8MA7SF8Y5P).
+ * - devMode: In dev mode FullStory won't record sessions. Any calls to SDK methods will `console.warn` that FullStory is in `devMode`. Defaults to 'false'.
```
Comment (patrick-fs) [inline src/index.d.ts:11]:
```suggestion
 * - devMode: In dev mode FullStory won't record sessions. Any calls to SDK methods will `console.warn` that FullStory is in `devMode`. Defaults to `false`.
```

**Review by patrick-fs — APPROVED** [review]
LGTM! I made a few small tweaks in some places.

**Comment by patrick-fs** [comment]
It looks like the branch build is failing on a few moderate `npm audit` issues. We'll get those fixed up soon and then this PR can land.

**Comment by patrick-fs** [comment]
@iuyt9003 would you please rebase your branch onto the latest `main`? This will get the updates required for CI to pass. Once done, I can approve and merge.

**Comment by iuyt9003** [comment]
No problem I am opening a new PR :)

**Pull request closed without merging**